test/Chainweb/Test/Pact/PactInProcApi.hs

```diff
@@ -284,7 +286,7 @@ minerKeysetTest :: TestBlockDb -> IO (IORef MemPoolAccess) -> WebPactExecutionSe
 minerKeysetTest bdb _mpRefIO pact = do
 
   -- run past genesis, upgrades
-  forM_ [(1::Int)..3] $ \_i -> runCut' noMiner
+  forM_ [(1::Int)..24] $ \_i -> runCut' noMiner
```

Comment:
why so high? this makes the test slower (applies to change in Version too obviously)

Reply:
It was making a test fail. 

The same thing that was causing the intermittent test failure for the pact 4.2.0 release wrt gas that I found with @LindaOrtega , there's a set of tests that rely on block height and the fork being applied too early (similar to when coin contract fork happened @ a low block height) makes some tests fail.